src/components/shared/UserMention/UserMention.tsx

```diff
@@ -29,51 +28,39 @@ interface Props {
 }
 
 const UserMention = ({
-  username,
+  user,
```

Comment:
I guess this could be destructured as well, but it's more of a nice to have rather than an objective change.